packages/host/app/services/realm-server.ts

```diff
@@ -197,8 +199,19 @@ export default class RealmServerService extends Service {
     });
 
     if (!response.ok) {
+      let responseText = await response.text();
+
+      // Temporary development instruction to help with user setup
+      let isDevelopment = config.environment === 'development';
+      if (isDevelopment && responseText.includes('User in JWT not found')) {
+        console.error(
+          '\x1b[1m\x1b[31m%s\x1b[0m',
```

Comment:
Prints an emphasized error message. This is just temporary until all devs run `pnpm register-all` again.

<img width="586" height="280" alt="Image" src="https://github.com/user-attachments/assets/8271834a-69bd-44f0-973f-6bae318b2219" />